src/fcs.jl

```diff
@@ -49,3 +49,42 @@ function pcap_has_fcs(reader::PcapReader)
     end
 end
 
+function _has_fcs(record::PcapRecord)
+    hdr = record.header
+    # In case of min-length packets we can't be certain whether they have FCS or not.
+    hdr.orig_len <= ETHERNET_MIN_LENGTH && return true
```

Comment:
need to make some assumption here - defaulting to true

Reply:
If we're going to return a default (rather than nothing/missing), then default to false. That's also what we do for unknown protocol types further down, so it must at least be consistent.